target/linux/realtek/files-6.12/drivers/net/dsa/rtl83xx/dsa.c

```diff
@@ -1936,7 +1938,19 @@ static int rtl83xx_port_mirror_add(struct dsa_switch *ds, int port,
 	pr_debug("Using group %d\n", group);
 	mutex_lock(&priv->reg_mutex);
 
-	if (priv->family_id == RTL8380_FAMILY_ID) {
+	if (priv->family_id == RTL9300_FAMILY_ID || priv->family_id == RTL9310_FAMILY_ID) {
+		/* enable port based mirror to specific port */
+		ctrl_reg_val = (mirror->to_local_port << 9) | 1;
+
+		/* set special vlan mirror mode */
+		ctrl_reg_val |= (1 << 5);
+
+		/* Direction of traffic to be mirrored */
+		if (!ingress)
```

Comment:
no, it isnt (as you showed yourself when you posted the snippet from the datasheet).